blog/_posts/2024-05-14-core-extension.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Introducing Core Kubernetes Extensions for AKS"
+description: "Learn what are core Kubernetes extensions, and how that extends functionality of your AKS clusters"
+date: 2025-05-14 # date is important. future dates will not be published
+author: Jane Guo # must match the authors.yml in the _data folder
+categories: 
+- extensions 
+- add-ons
+# - general
+# - operations
+# - networking
+# - security
+# - developer
+# - add-ons
+# - extensions
+---
+
+## Introducing Core Kubernetes Extensions
+
+### What Are Kubernetes Extensions?
+
+[Kubernetes extensions](https://learn.microsoft.com/en-us/azure/aks/cluster-extensions?tabs=azure-cli) (or cluster extensions) are pre-packaged applications that simplify the installation and lifecycle management of Azure capabilities on Kubernetes clusters. Examples include [Azure Backup](https://learn.microsoft.com/en-us/azure/backup/azure-kubernetes-service-backup-overview), [GitOps (Flux)](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-gitops-flux2), and [Azure Machine Learning](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-attach-kubernetes-anywhere?view=azureml-api-2). Third-party extensions (or Kubernetes apps), such as [Datadog AKS Cluster Extension](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/datadog1591740804488.dd_aks_extension?tab=Overview) and [Isovalent Cilium Enterprise](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/isovalentinc1662143158090.isovalent-cilium-enterprise?tab=Overview), are also available in the Azure Marketplace.
```

Comment:
can we remove 'en-us' part throughout doc for better localization support on docs?